### DISCUSSION TITLE
Chia CLI plot issues

### DISCUSSION THREAD

**Discussion by leoliyuanduo** [Chia-Network/chia-blockchain#2712] [Q&A]
I am using Chia CLI to plot
'chia plots create -k 32 -f farmKey -p poolKey -t D:\temp1\ -d D:\dist\ -u 128 -e'
It ended up getting a 101GB file named 'plot-k32-date-xxxxx.plot.2.tmp' in 'both temp1' and 'dist'
What can I do? Can I rename it to .plot?

**Comment by firmicus** [comment]
you should wait a bit ... I guess.

**Reply by leoliyuanduo** [reply]
You are right. It eventually renames the temp file
